docker-compose.yml

```diff
@@ -59,7 +59,7 @@ services:
   diagnostics:
     image: nebraltd/hm-diag:b30bd8a
```

Comment:
Can we put the diag version in here too? Or is that confusing?

Reply:
Yes I already did that as part of testing hm-diag#244. I can put one here too but please see the comment about balena not supporting .env file.